design/API.yml

```diff
@@ -328,6 +328,171 @@ components:
         - id
         - name
         - intro
+    Insight:
+      type: object
+      description: 灵感
+      properties:
+        id:
+          type: integer
+          description: ID
+          format: int64
+        content:
+          type: string
+          description: 内容
+        author:
+          $ref: "#/components/schemas/User"
+        created_at:
+          type: integer
+          description: 创建时间
+          format: int64
+        updated_at:
+          type: integer
+          description: 最后更新时间
+          format: int64
+        is_like:
+          type: boolean
+          description: 当前用户是否点赞
+        like_count:
+          type: integer
+          description: 点赞数量
+        comment_count:
+          type: integer
+          description: 评论数量
+        view_count:
+          type: integer
+          description: 浏览数量
+        medias:
+          type: array
+          description: 媒体文件，如果没有则为空数组
+          items:
+            $ref: "#/components/schemas/Media"
+        attachment:
+          $ref: "#/components/schemas/Attachment"
+      required:
+        - id
+        - content
+        - author
+        - created_at
+        - updated_at
+        - is_like
+        - like_count
+        - comment_count
+        - view_count
+        - medias
+    Attachment:

```

Comment:
These attachments are all hosted in rich text editor, which is represented by a huge json. So there is no need to return these metadatas seprately.

Reply:
The backend only need to return the json text untouchedly.